auth/grpctransport/grpctransport.go

```diff
@@ -16,12 +16,13 @@ package grpctransport
 
 import (
 	"context"
+	"crypto/tls"
 	"errors"
 	"fmt"
 	"net/http"
 
 	"cloud.google.com/go/auth"
-	"cloud.google.com/go/auth/credentials"
+	detect "cloud.google.com/go/auth/credentials"
```

Comment:
there seems to be no need to rename the import, I think we should prefer the normal name.

Reply:
Reverted to original, but note that "httptransport.go" does rename this to detect so I was simply trying to make the 2 consistent - you guys can figure it out later : )